app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminClickSaveOnProductsMassAttributeUpdateActionGroup.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminClickSaveOnProductsMassAttributesUpdateActionGroup">
+        <annotations>
+            <description>Clicks on 'Save' button on products mass attributes update page.</description>
+        </annotations>
+
+        <click selector="{{AdminEditProductAttributesSection.Save}}" stepKey="save"/>
+
+    </actionGroup>
+</actionGroups>
```

Comment:
Preserve an empty row at the end of the file.
```suggestion
</actionGroups>

```

Reply:
Empty row has been added, thanks.